### PR TITLE
chore: monitor infura

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: monitoring/infura.spec.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  // wait to be logged in
+  await Selector("[data-testid='header-user-icon']");
+
+  await t.navigateTo("https://infura.io/dashboard/ethereum").addRequestHooks(statsLogger);
+
+  // wait to see the create project button
+  await Selector("[data-testid='create-project']");
+
+  const requestCount = await getRequestCount(statsLogger);
+  const maximumRequest = 1000000;
```
Comment (rjchow) [inline monitoring/infura.spec.ts:71]: where are the numbers from?
maximumRequest
thresholdPercentage

File: monitoring/infura.spec.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  const thresholdPercentage = 60; // 60%
+  const threshold = (maximumRequest * thresholdPercentage) / 100;
+  if (requestCount > threshold) {
+    await openGithubIssue(requestCount);
```
Comment (rjchow) [inline monitoring/infura.spec.ts:75]: should it also make a github issue if the test fails to complete?

File: monitoring/infura.spec.ts (hunk 1 above)
Comment (Nebulis) [reply monitoring/infura.spec.ts:71]: maximumRequest => infura plan
threshold => just a decision

File: monitoring/infura.spec.ts (hunk 2 above)
Comment (Nebulis) [reply monitoring/infura.spec.ts:75]: I wondered actually. I will see what happens if it fails, we should see something maybe.

File: monitoring/infura.spec.ts (hunk 1 above)
Comment (rjchow) [reply monitoring/infura.spec.ts:71]: comment? :D

**Review by rjchow — APPROVED** [review]

**Pull request merged**